tests/functional/pyocf/types/ctx.py

```diff
@@ -51,10 +54,29 @@ def __init__(self, lib, name, logger, data, cleaner):
         if result != 0:
             raise OcfError("Context initialization failed", result)
 
+        if self.default is None or self.default() is None:
```

Comment:
It's a weakref - the way you use it is that you call it and you get either reference to object or None if object was already destroyed. Default is a weakref because it has static lifetime (class members are instantiated at import and finalized at interpreter shutdown).